boot/image-sig.c

```diff
@@ -17,7 +17,7 @@ DECLARE_GLOBAL_DATA_PTR;
 #define IMAGE_MAX_HASHED_NODES		100
 
 struct checksum_algo checksum_algos[] = {
-#if !CONFIG_IS_ENABLED(FIT_SIGNATURE_STRICT)
+#if CONFIG_IS_ENABLED(SHA1) && !CONFIG_IS_ENABLED(FIT_SIGNATURE_STRICT)
```

Comment:
Other options can be make the options FIT_SIGNATURE_STRICT depends on CONFIG_IS_ENABLED.

Reply:
@quaresmajose FIT_SIGNATURE_STRICT just disables using some weak algos like SHA1 for checking image signatures. It doesn't restrict using SHA1 in other parts of u-boot.